producer-cpp/docker-rtsp/Dockerfile

```diff
@@ -32,6 +32,22 @@ WORKDIR /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/
 RUN cmake .. -DBUILD_GSTREAMER_PLUGIN=ON &&\
```

Comment:
Although there is no change to this line as part of this PR, can we include a space between && and \